sql/plan/exchange_test.go

```diff
@@ -150,6 +150,10 @@ func (partitionable) Schema() sql.Schema {
 
 func (partitionable) Name() string { return "partitionable" }
 
+func (partitionable) IsTemporary() bool {
```

Comment:
From the dolt PR, you should define a new TemporaryTable interface extension to Table interface so that not every table implementation needs to provide IsTemporary